index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>personal portfolio</title>
+</head>
+
+<body>
+    <div class="container">
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, which is also part of the project requirements, you are suppose to use a `<section> ` tag instead of a `<div>`. This is because `<section>` tags makes your code more semantic which is good for assistive technologies(like screen readers) and search engine optimization. `div` are generic tag. Checkout this [link](https://www.w3schools.com/html/html5_semantic_elements.asp). It all about best practices.